pipeline/psi-only-executor.groovy

```diff
@@ -100,13 +102,22 @@ node(nodeName) {
 
     stage('Publish Results') {
         /* Publish results through E-mail and Google Chat */
-
-        sharedLib.sendGChatNotification(testResults, tierLevel.capitalize())
+        if(ciMap["pipeline"].containsKey("tags")){
+                def tag = ciMap["pipeline"]["tags"]
+                def tags_list = tag.split(',') as List
+                def stage_index = tags_list.findIndexOf { it ==~ /stage-\w+/ }
+                stageLevel = tags_list.get(stage_index)
+                run_type = ciMap["pipeline"]["run_type"]
+                }
```

Comment:
Can you please take care of indentation here.

Reply:
Done